src/Json.Schema.ToDotNet.UnitTests/DataModelGeneratorTests.cs

```diff
@@ -2363,7 +2369,152 @@ public int GetHashCode(C obj)
                 ["D"] = new ExpectedContents()
             };
 
-            Assert.FileContentsMatchExpectedContents(_testFileSystem, expectedContentsDictionary);
+            VerifyGeneratedFileContents(expectedContentsDictionary);
+        }
+
+        [Fact(DisplayName = "DataModelGenerator generates attributes for properties with defaults.")]
+        public void GeneratesAttributesForPropertiesWithDefaults()
+        {
+            const string Schema =
+@"{
+  ""type"": ""object"",
+  ""properties"": {
+    ""intProp"": {
```

Comment:
>intProp [](start = 6, length = 7)

why so stingy with your identifiers? Is there anything wrong with 'integerProperty' here? readability of these string literals honestly would be improved. the double-quotes work against the terseness of your ids. fine as a TBD, of course, if you even decide to take the note

Reply:
I don't mind changing them. Doing it now...

---
In reply to: [244508136](https://github.com/Microsoft/jschema/pull/62#discussion_r244508136) [](ancestors = 244508136)

Reply:
ugh. sorry to put you through this. wish you had an automated process. :) i can't resist asking, how do you handle the double quote escaping? do you have any file replace magic for this? 

---
In reply to: [244508723](https://github.com/Microsoft/jschema/pull/62#discussion_r244508723) [](ancestors = 244508723,244508136)

Reply:
No, I just type the double-quotes. Anyway, this one is now fixed.

---
In reply to: [244510035](https://github.com/Microsoft/jschema/pull/62#discussion_r244510035) [](ancestors = 244510035,244508723,244508136)